generated/nidigital/setup.py

```diff
@@ -51,7 +51,7 @@ def read_contents(file_to_read):
     tests_require=['pytest'],
     test_suite='tests',
     classifiers=[
-        "Development Status :: 3 - Alpha",
+        "Development Status :: 4 - Beta",
```

Comment:
I think we should keep this in Alpha still. Beta once we think the API is stable.

Reply:
Fixed